docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,157 @@
+1. Enter a URL
+1. Click **Submit** to add the link. The link will appear listed under the Grafana News heading.
+To vote for a link, click the triangle icon next to the name of the link.
+
+## Log in to Grafana
+
+Besides being an open-source observability tool, Grafana has its own built-in alerting service.   This means that you can receive notifications whenever there is an event of interest in your data, and even see these events graphed in your visualizations.
+
+In your browser, simply navigate to [localhost:3000](http://localhost:3000). You should get logged in automatically
+
+## Create an Alert
+
+Next, we'll establish an alert within Grafana Alerting to notify us whenever our sample app experiences a specific volume of requests.
+
+In Grafana, toggle the menu at the top left side of the screen, and **navigate to Alerting** > **Alert rules**. Click on  **+ New alert rule**.
+
+## Enter alert rule name
+
+1. Make it short and descriptive as this will appear in your alert notification. For instance, server-requests-duration
+
+## Define query and alert condition
+
+In this section we define the conditions that trigger alerts. 
+
+1. **Choose the data source** from the drop-down menu. **Select Prometheus**.
+
+    {{< admonition type="note" >}}
+    To visualize this data in Grafana, we need time-series metrics that we can collect and store. We can do that with [Prometheus](https://grafana.com/docs/grafana/latest/getting-started/get-started-grafana-prometheus/), which is pulling metrics from our sample app.
+    {{< /admonition >}}
+
+1. In the Query editor, **switch to Code mode**.
+1. Paste the query tns_request_duration_seconds_count{status_code="200",method="GET"}
+You should see the number of requests made to the server that had a response code 200.
+1. Keep expressions “B” and “C” as they are. These expressions (Reduce and Threshold, respectively) come by default when creating a new rule. Expression “B”, selects the last value of our query “A”, while the Threshold expression “C” will check if the last value from expression “B” is above a specific value. In addition, the Threshold expression is the alert condition by default. Enter 0.2 as threshold value. You can read more about queries and conditions [here](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/queries-conditions/#expression-queries).
+1. Click Preview to run the queries.
+
+At this point, our alert should be working (it should be either in [Firing or Normal state](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/view-state-health/#alert-rule-state)). If it returns an error, follow the instructions in the error message. If it returns “No data,” reselect the metrics from the Metrics browser; make sure the Input of each expression is correct (expression B: Input A and expression C: Input B). If you still get stuck, you are very welcome to post questions in our [Grafana Community forum](https://community.grafana.com/).
```

Comment:
do you mean that your alert rule will start returning data?

Reply:
good point. i think it is more clear to say that will start returning data instead of "should be working). Plus talking about firing or normal state could be removed. I would keep the last part, in case users get 'no data' Wdyt@brendamuir ?

Reply:
ok!